### PR TITLE
docs: Fix a couple of minor typos in `rclone_mount.md`

### PR DESCRIPTION
**Pull request by peterkaminski**
Changes:

* "transferring" instead of "transfering"
* "connection" instead of "connnection"
* "mount" instead of "mount mount"

#### What is the purpose of this change?

I happened to notice a typo in `rclone_mount.md`, and I ran spellcheck and found a couple more.

I did not expand the scope to check any other docs files.

#### Was the change discussed in an issue or in the forum before?

No, not previously discussed.

#### Checklist

- [ y] I have read the [contribution guidelines](https://github.com/ncw/rclone/blob/master/CONTRIBUTING.md#submitting-a-pull-request).
- [ n/a] I have added tests for all changes in this PR if appropriate.
- [ n/a] I have added documentation for the changes if appropriate.
- [ y] All commit messages are in [house style](https://github.com/ncw/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [ y] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

File: docs/content/commands/rclone_mount.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ systems are a long way from 100% reliable. The rclone sync/copy
 commands cope with this with lots of retries.  However rclone mount
 can't use retries in the same way without making local copies of the
 uploads. Look at the [file caching](#file-caching)
-for solutions to make mount mount more reliable.
```
Comment (ncw) [inline docs/content/commands/rclone_mount.md]: This needs to be changed in `cmd/mountlib/mount.go`  - `rclone_mount.md` is auto generated from there.

File: docs/content/commands/rclone_mount.md [hunk 2]
```diff
@@ -391,8 +391,8 @@ rclone mount remote:path /path/to/mountpoint [flags]
       --crypt-password2 string                     Password or pass phrase for salt. Optional but recommended.
       --crypt-remote string                        Remote to encrypt/decrypt.
       --crypt-show-mapping                         For all files listed show how the names encrypt.
-      --delete-after                               When synchronizing, delete files on destination after transfering (default)
-      --delete-before                              When synchronizing, delete files on destination before transfering
+      --delete-after                               When synchronizing, delete files on destination after transferring (default)
```
Comment (ncw) [inline docs/content/commands/rclone_mount.md]: These need to be changed in `fs/config/configflags/configflags.go`

File: docs/content/commands/rclone_mount.md [hunk 3]
```diff
@@ -503,7 +503,7 @@ rclone mount remote:path /path/to/mountpoint [flags]
       --pcloud-client-secret string                Pcloud App Client Secret
   -P, --progress                                   Show progress during transfer.
       --qingstor-access-key-id string              QingStor Access Key ID
-      --qingstor-connection-retries int            Number of connnection retries. (default 3)
+      --qingstor-connection-retries int            Number of connection retries. (default 3)
```
Comment (ncw) [inline docs/content/commands/rclone_mount.md]: And this one is in `backend/qingstor/qingstor.go`

**Comment by peterkaminski** [comment]
Okay, done!  I've changed the original files the same way.  Sorry - I _did_ grep the `docs` directory to see if those changes occurred upstream somehow, but I didn't think to grep the code, too. D'oh!

I did not back out the changes in `docs/content/commands`, I hope that's okay.

**Comment by ncw** [comment]
Thanks for doing that - I'll merge it now :-)

There are autogenerated markers in the files `###### Auto generated by spf13/cobra on 15-Oct-2018` but they are at the end and easy to miss - sorry!

**Pull request merged**